LibreNMS/Util/ModuleTestHelper.php

```diff
@@ -249,7 +249,13 @@ public static function findOsWithData($modules = [])
             [$os, $variant] = self::extractVariant($file);
 
             // calculate valid modules
-            $data_modules = array_keys(json_decode(file_get_contents($file), true));
+            $decoded = json_decode(file_get_contents($file), true);
+
+            if (json_last_error()) {
+                throw new \Exception("Unable to decode file $file");
+            }
+
+            $data_modules = array_keys($decoded);
 
             if (json_last_error()) {
                 echo "Invalid json data: $base_name\n";
```

Comment:
Remove?

Reply:
I wan't to keep it in.  Otherwise we have no clue which file has invalid json.

Reply:
But you have 2 of them. The old one and the new one you added

Reply:
oh, heh